casdoorsdk/base.go

```diff
@@ -24,6 +24,27 @@ import (
 	"strings"
 )
 
+var (
+	// globalClient is a shared http Client.
+	globalClient = NewHTTPClient()
+)
+
+// SetCustomHTTPClient sets custom http Client.
+func SetCustomHTTPClient(s Doer) {
```

Comment:
Doer --> HttpClientInterface

Reply:
@hsluoyz It's already an interface. I think it should be `HttpClient`.